buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/PrecommitTasks.groovy

```diff
@@ -123,8 +123,14 @@ class PrecommitTasks {
         project.tasks.withType(CheckForbiddenApis) {
             dependsOn(buildResources)
             targetCompatibility = project.runtimeJavaVersion >= JavaVersion.VERSION_1_9 ?
-                    project.runtimeJavaVersion.getMajorVersion() :
-                    project.runtimeJavaVersion
+                    project.runtimeJavaVersion.getMajorVersion() : project.runtimeJavaVersion
+            if (project.runtimeJavaVersion > JavaVersion.VERSION_11) {
+                doLast {
+                    println "Forbidden APIs does not support java version past 11. Will use the signatures from 11 for " +
```

Comment:
Why a bare print statement? Seems this should at least be a warning log. But what can a dev do about it? Maybe nothing should be printed and an issue created and linked here to remove once forbidden apis supports Java 12.

Reply:
I agree, i converted it to info so it leaves a trace in CI. 
An issue isn't actionable either unless  we implement the support in forbidden APIs ourselves.